wp-content/themes/pub/wporg-learn-2024/patterns/sidebar-meta-info.php

```diff
@@ -5,37 +5,81 @@
  * Inserter: no
  */
 
- use function WPOrg_Learn\Sensei\{get_my_courses_page_url}
+use function WPOrg_Learn\Sensei\{get_my_courses_page_url};
+$current_post = get_post();
+$current_post_type = get_post_type();
 
 ?>
 
 <!-- wp:group {"align":"full","className":"wporg-learn-sidebar-meta-info","layout":{"type":"constrained"}} -->
 <div class="wp-block-group alignfull wporg-learn-sidebar-meta-info">
 
-	<?php if ( Sensei_Course::is_user_enrolled( get_the_ID() ) ) : ?>
-	<!-- wp:paragraph {"style":{"elements":{"link":{"color":{"text":"var:preset|color|blueberry-1"}}},"typography":{"fontStyle":"normal","fontWeight":"400","lineHeight":26px}},"textColor":"blueberry-1","fontSize":"normal","fontFamily":"inter","className":""wporg-learn-sidebar-all-courses"} -->
-	<p class="has-blueberry-1-color has-text-color has-link-color has-inter-font-family has-normal-font-size wporg-learn-sidebar-all-courses" style="font-style:normal;font-weight:400;line-height:26px">
-		<a href="<?php echo esc_url( get_my_courses_page_url() ); ?>">
-			<?php esc_html_e( 'All My Courses', 'wporg-learn' ); ?>
-		</a>
-	</p>
-	<!-- /wp:paragraph -->
-	<?php endif; ?>
+	<?php if ( 'course' === $current_post_type ) : ?>
```

Comment:
I think rather than branching we should create new patterns for each page sidebar; `sidebar-course.php`, sidebar-lesson-plan.php`. It will make it easier to find the code for each, and simplify the imports, remove logic, etc.

Reply:
65e27f2